ispdb/cek.ne.jp.xml

```diff
@@ -4,18 +4,27 @@
     <domain>cek.ne.jp</domain>
     <displayName>エコーシティー・駒ケ岳</displayName>
     <displayShortName>CEK-Net</displayShortName>
+    <incomingServer type="imap">
+      <hostname>imap.cek.ne.jp</hostname>
+      <port>993</port>
+      <socketType>SSL</socketType>
+      <username>%EMAILLOCALPART%</username>
```

Comment:
issue: https://faq.cek.co.jp/CEK%E3%81%AE%E3%83%A1%E3%83%BC%E3%83%AB%E3%82%92%E4%BD%BF%E3%81%86%E6%96%B9%E6%B3%95-64b667d02407c8001bfabc95 says (after translating through DeepL):

> [User name or account name] Email address in the User Guide Notice (common for POP3/IMAP4)

Which I understand to mean that we should use the whole email address, not just the local part.

Same comment applies to POP3 and SMTP.